src/screens/Ticketing/Purchase/Payment/SelectPaymentMethod/index.tsx

```diff
@@ -374,20 +378,30 @@ const useStyles = StyleSheet.createThemeHook((theme) => ({
   heading: {
     flex: 1,
     flexDirection: 'row',
-    justifyContent: 'center',
+    justifyContent: 'flex-start',
     paddingBottom: 24,
```

Comment:
Ser at her er mange hardkodet verdier for padding o.l. Disse burde kanskje byttes ut med verdier fra `themes.spacings`? 

```suggestion
    paddingBottom: theme.spacings.xLarge,
```

Oversikt over verdiene ligger her: https://github.com/AtB-AS/design-system/blob/main/packages/theme/src/sizes.ts

Reply:
Holder forresten på med å implementere komponenter for samme typen radio buttons i #1622 nå, så da ser jeg for meg at det kan være fint å oppdatere dette til å bruke samme komponenter. Altså ikke nødvendig å forandre noe på radio buttons, siden de blir byttet ut senere.

Reply:
Takk for linking til themes 🤷‍♂️ De visste jeg ikke om 😁